Sources/Clappr/Classes/Plugin/Core/MediaControl/MediaControl.swift

```diff
@@ -0,0 +1,236 @@
+            listenTo(container,
+                     eventName: Event.disableMediaControl.rawValue) { [weak self] _ in self?.hide() }
+        }
+    }
+
+    private func bindPlaybackEvents() {
+        if let playback = activePlayback {
+            listenTo(playback, eventName: Event.ready.rawValue) { [weak self] _ in
+                self?.showControls = true
+                self?.show { [weak self] in
+                    self?.disappearAfterSomeTime()
+                }
+            }
+
+            listenTo(playback, eventName: Event.didComplete.rawValue) { [weak self] _ in
+                self?.hide()
+            }
+
+            listenTo(playback, eventName: Event.playing.rawValue) { [weak self] _ in
+                self?.show { [weak self] in
+                    self?.disappearAfterSomeTime()
+                }
+            }
+
+            listenTo(playback, eventName: Event.didPause.rawValue) { [weak self] _ in
+                self?.keepVisible()
+            }
+
+            listenTo(playback, eventName: Event.error.rawValue) { [weak self] _ in
+                self?.showControls = false
+            }
+        }
+    }
+
+    func show(animated: Bool = false, completion: (() -> Void)? = nil) {
+        let duration = animated ? animationDuration : 0
+
+        if self.alpha == 0 {
+            self.isHidden = false
+        }
+
+        UIView.animate(
+            withDuration: duration,
+            animations: {
+                self.alpha = 1
+            },
+            completion: { _ in
+                self.isHidden = false
+                completion?()
+            }
+        )
+    }
+
+    func hide(animated: Bool = false, completion: (() -> Void)? = nil) {
+        if !alwaysVisible {
+            let duration = animated ? animationDuration : 0
+
+            UIView.animate(
+                withDuration: duration,
+                animations: {
+                    self.alpha = 0
+            },
+                completion: { _ in
+                    self.isHidden = true
+                    completion?()
+            }
+            )
+        }
+    }
+
+    private func disappearAfterSomeTime(_ duration: TimeInterval? = nil) {
+        hideControlsTimer?.invalidate()
+        hideControlsTimer = Timer.scheduledTimer(timeInterval: duration ?? secondsToHideControlFast,
+                                                 target: self, selector: #selector(MediaControl.hideAndStopTimer), userInfo: nil, repeats: false)
+    }
+
+    private func keepVisible() {
+        hideControlsTimer?.invalidate()
+    }
+
+    @objc func hideAndStopTimer() {
+        hideControlsTimer?.invalidate()
+        hide(animated: true)
+    }
+
+    @objc func tapped() {
+        hideAndStopTimer()
+    }
+
+    override func render() {
+        self.isHidden = true
+        view = UIView()
+        view.backgroundColor = UIColor.clapprBlack60Color()
```

Comment:
The Clappr `MediaControl` don't has a translucid black background. Instead, it's a light gradient at the bottom.

You can see here: http://clappr.io/demo/

Plus, we can also check if there's any way to customize this.

Reply:
We know clappr's UI is different then steve, however if we want to keep the same looks as now we'd have to invest much more time. We thought that we agreed to make it simple: make steve's UI and just change assets. We can discuss this with the team with u want to = ) 
But for now I would suggest to move on and create another card to adjust clappr's UI if needed.